services/github/github-commit-activity.service.js

```diff
@@ -23,16 +23,17 @@ module.exports = class GithubCommitActivity extends LegacyService {
   static get route() {
     return {
       base: 'github/commit-activity',
-      pattern: ':interval(y|4w|w)/:user/:repo',
+      pattern: ':interval(y|m|4w|w)/:user/:repo',
     }
   }
 
   static get examples() {
     return [
       {
-        title: 'GitHub commit activity the past year',
-        pattern: 'y/:user/:repo',
-        namedParams: { user: 'eslint', repo: 'eslint' },
+        title: 'GitHub commit activity',
+        // Override the pattern to omit the deprecated interval "4w".
+        pattern: ':interval(y|m|w)/:user/:repo',
+        namedParams: { interval: 'm', user: 'eslint', repo: 'eslint' },
         staticPreview: {
           label: 'commit activity',
           message: '457/year',
```

Comment:
With the new dropdown support, should we make the previews consistent with the selection (for example here using `457/m` to be consistent with the named `interval`) or make them different (as they are here) to indicate multiple options?

Reply:
Oops! I tried to keep them the same. Feel free to fix!

Reply:
Done!